packages/concerto-cli/index.js

```diff
@@ -72,7 +72,7 @@ require('yargs')
                 .then((result) => {
                     Logger.info('Input is valid');
                     if (!options.functional) {
-                        Logger.info(result);
+                        console.log(result);
```

Comment:
Intentional?

Reply:
Yes, I should have noted this as an unrelated change. `Logger` adds a log of formatting and metadata which makes the output difficult to pipe to another command or to stream to a file.

I made this decision for the new `generate` command, then decided to standardise the output of similar commands.